src/App.js

```diff
@@ -1,25 +1,36 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { useEffect, useState } from "react";
+import './App.css';
+import { useEffect, useState } from 'react';
 
 function App() {
-  const [data, setData] = useState(null)
+  const [data, setData] = useState({});
   useEffect(() => {
     async function loadData() {
-      const response = await fetch('https://redi-final-restaurants.herokuapp.com/restaurants')
-      const data = await response.json()
-      setData(data)
+      const response = await fetch(
+        'https://redi-final-restaurants.herokuapp.com/restaurants'
+      );
+      const data = await response.json();
+      setData(data);
     }
-    loadData()
-  }, [])
-  console.log(data)
+    loadData();
+  }, []);
+  console.log(data);
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>Hello Team 1!</h1>
-      </header>
+    <div className='App'>
+      {data.results.map((rest) => (
+        <div>
+          <h3>{`Name: ${rest.name}`}</h3>
+          <h4>{`Cuisine: ${rest.cuisine} `}</h4>
+          <h5>{rest.formatted_address}</h5>
+          <h5>{`Email: ${rest.social.email}  || Phone: ${rest.social.phone}`}</h5>
+          <img src={rest.photos[0].links[0]} alt='icon' />
+          <h6>{`Price level ${rest.price_level} ||  Rating ${rest.rating}`}</h6>
+          <div>{rest.opening_hours.open_now ? 'open' : 'closed'} </div>
+          <div>{rest.pickup ? 'pickup' : null} </div>
```

Comment:
you can choose to not render the div at all if you have null value like this
```suggestion
          {rest.pickup  && <div>pickup </div>}
```